src/d3d9/d3d9_depth_bias.cpp

```diff
@@ -0,0 +1,200 @@
+#include "d3d9_depth_bias.h"
+#include "d3d9_find_depth_bias_factor.h"
+#include "../dxvk/dxvk_format.h"
+#include "vulkan/vulkan_core.h"
```

Comment:
Vs code moment

Reply:
indeed

Reply:
A little offtopic, but I noticed similar moment in [gamescope code](https://github.com/Plagman/gamescope/commit/1b34a570a51d0a65451d1f2a990487add8cde40a#diff-0ca179f8adb820354aba0b6a063350f9e32689950ead39e8bbeca021fa8ba0e0)

Reply:
Josh knows what he's talking about. :)